### PR TITLE
[Win] Process Key and Mouse event

### PR DESCRIPTION
**Pull request by Mon1Amour4**

### CODE REVIEW

**Review by LtSnail — APPROVED** [review]

**Comment by Mon1Amour4** [comment]
D:\a\Fuego\Fuego\Engine\Fuego\Events\MouseEvent.h(76,62): warning C4244: 'argument': conversion from 'const int' to 'Fuego::MouseCode', possible loss of data [D:\a\Fuego\Fuego\build\win\Engine\FuegoEngine.vcxproj]
  (compiling source file '../../../Engine/Fuego/Windows/WindowWin.cpp')
  
  It's because Ir Ran tests before merge of the fix

File: Engine/fupch.h [hunk 1]
```diff
@@ -20,5 +20,5 @@
 #import <simd/simd.h>
 #endif
 #elif FUEGO_PLATFORM_WIN
-#include <windows.h>
+#include <windowsx.h>
```
Comment (LtSnail) [inline Engine/fupch.h:23]: Why did you change the header?

Comment (Mon1Amour4) [reply Engine/fupch.h:23]: X version contains some useful methods for gettting X\Y position of your mouse
